tests/test_message.py

```diff
@@ -1,7 +1,33 @@
 import pytest
 import linkatos.message as message
 
+
+# test link detection
+
 def test_contains_a_https_url():
     expected = "https://example.org"
 
-    assert message.contains_a_link("foo https://example.org") == expected
+    assert message.extract_url("foo https://example.org") == expected
+
+
+def test_contains_a_http_url():
+    expected = "http://example.com"
+
+    assert message.extract_url("foo http://example.com bar") == expected
+
+
+def test_is_not_a_url():
+
+    assert message.extract_url("foo htts://example.org") is None
+
+
+# test yes detection
+
+def test_contains_a_yes():
+
+    assert message.has_a_yes("foo yes bar") is True
```

Comment:
you should add more situations where `yes` should be catched. E.g. `lorem ipsum yes.`, `and yes`, `yes or no`.